vllm/engine/arg_utils.py

```diff
@@ -394,6 +394,8 @@ class EngineArgs:
         str] = ModelConfig.logits_processor_pattern
 
     speculative_config: Optional[Dict[str, Any]] = None
+    draft_tensor_parallel_size: Optional[int] = None
+    no_auto_speculative: bool = False
```

Comment:
nit: maybe no_auto_speculators?

Reply:
Is `draft_tensor_parallel_size>2` supposed to work? I got an error saying not supported

Reply:
removed the arg to disable auto-detection and added a comment for draft_tensor_parallel_size